### PR TITLE
onelogin

### PR DESCRIPTION
**Pull request by dkim32**
# Pull Request/Issue Resolution

**Description of Change:**
Updating documentation for Onelogin to follow Azure AD format. 

**Reason for Change:**
To make it more intuitive for our customers to integrate their IdP. 
Additional screenshots. 
Updated Braze app. 

Closes #**ISSUE_NUMBER_HERE**

### Is this change associated with a Braze feature/product release?
- [ ] Yes (__Feature Release Date:__)
- [ ] No

> If yes, please note the date of the feature release.

---
---

## PR Checklist
- [ ] Ensure you have completed [our CLA](https://www.braze.com/docs/cla/).
- [ ] Tag @EmilyNecciai as a reviewer when the your work is _done and ready to be reviewed for merge_.
- [ ] Consult the [Docs Text Formatting Guide](https://github.com/Appboy/success/wiki/Docs-Text-Formatting-Guide).
- [ ] Consult the [Docs Writing Style Guide & Best Practices](https://github.com/Appboy/success/wiki/Writing-Style-Guide-&-Best-Practices).
- [ ] Tag others as Reviewers as necessary.
- [ ] If you have modified any links, be sure to add redirects to `config/nginx.conf.erb`.

## ATTENTION: REVIEWERS OF THIS PR
- [ ] Read our [Reviewing a PR page](https://github.com/Appboy/braze-docs/wiki/Reviewing-a-PR) for more on our reviewing suggestions.
- [ ] Preview all changes in the linked Heroku environment (click `View deployment` button below, then "Docs". A `502` error just means you should refresh until you see the Docs Home page.
---
---

### CODE REVIEW

**Comment by EmilyNecciai** [comment]
@dkim32 Hey! Is this something that should/could go out tomorrow?

**Comment by dkim32** [comment]
> @dkim32 Hey! Is this something that should/could go out tomorrow?

@EmilyNecciai this can go out with next week's major release. i'm just trying to get ahead. look out for pingid PR as well :)

File: _docs/_user_guide/administrative/access_braze/single_sign_on/onelogin.md [hunk 1]
```diff
@@ -5,53 +5,62 @@ page_order: 5
...
+
 ## Requirements
 
-- Braze account.
-- OneLogin account.
+Upon setup, you will be asked to provide a Sign-On URL and an Assertion Consumer Service (ACS) URL.  
+
+| Requirement | Details |
+|---|---|
+| **Braze Domain** | You will need your Braze domain to setup Braze within OneLogin. If your instance is `US-01`, you will need to input `dashboard-01.braze.com` into the OneLogin dashboard. |
```
Comment (papiya) [inline _docs/_user_guide/administrative/access_braze/single_sign_on/onelogin.md]: Would it be more clear if we said that "If your braze dashboard url is http://dashboard-01.braze.com, input dashboard-01.braze.com " ? I think the above language does not work for eu.

File: _docs/_user_guide/administrative/access_braze/single_sign_on/onelogin.md [hunk 2]
```diff
@@ -5,53 +5,62 @@ page_order: 5
...
+
 
-## Step 1: Configure OneLogin
+## Service Provider (SP) Initiated Login within OneLogin
+
+### Step 1: Configure the Braze App
 
 1. Log into [OneLogin](https://www.onelogin.com/learn/saml). Click on `Administration`.
```
Comment (papiya) [inline _docs/_user_guide/administrative/access_braze/single_sign_on/onelogin.md]: Is that the url for everyone ? The one that I used for development was https://braze-inc-dev.onelogin.com/portal/ .

Comment (dkim32) [reply _docs/_user_guide/administrative/access_braze/single_sign_on/onelogin.md]: we can update to the following url: https://app.onelogin.com/login

**Comment by EmilyNecciai** [comment]
@cla-bot check

**Pull request merged**